src/schemas/raster.schema.json

```diff
@@ -1,25 +1,25 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://github.com/hubmapconsortium/vitessce/#images",
+  "$id": "https://github.com/hubmapconsortium/vitessce/#rasters",
   "title": "Vitessce image data",
   "type": "object",
   "definitions": {
-    "image": {
+    "raster": {
       "type": "object",
-      "additionalProperties": false,
+      "additionalProperties": true,
```

Comment:
Can you keep additionalProperties false, and define the props you need?

Reply:
Trevor has an extra property for `zarr` temporarily.  I can file an issue to remove this once that is resolved.